_format/1.1/index.md

```diff
@@ -585,6 +596,96 @@ that includes a [profile alias][profile aliases]:
 > Note: Additional link types, similar to `profile` links, may be specified in
 the future.
 
+#### <a href="#document-links-link-relations" id="document-links-link-relations" class="headerlink"></a> Link relation types
+
+The following link relation types **MAY** be used as the value of the `rel` link
+parameter object member:
+
+  * `add`: The link's target points to a resource where a user agent can cause
+    an addition to the link's context.
+  * `update`: The link's target points to a resource where a user agent can cause
+    an update the link's context.
+  * `remove`: The link's target points to a resource where a user agent can cause
+    a removal of all or part of the link's context.
+  * `relate`: The link's target points to a resource where a user agent can cause
+    a direct or indirect relationship to be established with the link's context.
+
+> Note: These link relation types correspond to operations described by the
+JSON:API specification and it is expected that implementations will use them to
+reference resources that will process requests according to [the requirements for
+creating, updating or deleting resources](https://jsonapi.org/format/1.1/#crud). However, this is not a strict
+limitation on their use.
+
+In the example below, two links are established. A `self` link indicating the
+URL of the request that generated the current collection of articles and an `add`
+link indicating the that the client can add an article to this collection.
+
+```json
+"links": {
+  "self": {
+    "href": "http://example.com/articles",
+    "rel": ["self", "add"]
+  }
+}
+```
+
+In the next example, a `comment` link is a member of a `comments` [relationship
+object][relationships] and that relationship object is a member of an `article`'s relationships
+object. In this case, the `rel` member establishes two links indicating that
+the client can create a new comment at the target URL and that requests to the
+target `href` can create a relationship to the article's `comments` field.
+
+```json
+"links": {
+  "self": "http://example.com/articles/1/relationships/comments",
+  "related": "http://example.com/articles/1/comments",
+  "comment": {
+    "href": "http://example.com/articles/1/comments",
+    "rel": ["add", "relate"]
```

Comment:
So this is where the `relate` extension link relation type comes into play.

It's not crystal clear to me. I think this is to convey that the act of adding a new resource object to the collection would also result in a new relationship being established?